arangosh/Backup/BackupFeature.cpp

```diff
@@ -681,68 +681,71 @@ void BackupFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opti
                      static_cast<std::underlying_type<Flags>::type>(Flags::Hidden));
 
   options->addOption("--allow-inconsistent",
-                     "whether to attempt to continue in face of errors (may "
-                     "result in inconsistent backup state)",
+                     "whether to attempt to continue in face of errors. "
```

Comment:
to be consistent, i'd suggest using semicolon here to or a dot below. my choice would be ... errors; may ...